java/src/com/mongodb/csfle/CSFLEHelpers.java

```diff
@@ -0,0 +1,219 @@
+package com.mongodb.csfle;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Base64;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.bson.BsonBinary;
+import org.bson.BsonDocument;
+import org.bson.Document;
+import org.bson.conversions.Bson;
+
+import com.mongodb.AutoEncryptionSettings;
+import com.mongodb.ClientEncryptionSettings;
+import com.mongodb.ConnectionString;
+import com.mongodb.MongoClientSettings;
+import com.mongodb.client.MongoClient;
+import com.mongodb.client.MongoClients;
+import com.mongodb.client.MongoCollection;
+import com.mongodb.client.model.IndexOptions;
+import com.mongodb.client.model.vault.DataKeyOptions;
+import com.mongodb.client.vault.ClientEncryption;
+import com.mongodb.client.vault.ClientEncryptions;
+
+/*
+ * Copyright 2008-present MongoDB, Inc.
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * Helper methods and sample data for this companion project.
+ */
+public class CSFLEHelpers {
+
+    // Sample data
+    public static String SAMPLE_NAME_VALUE = "John Doe";
+    public static Integer SAMPLE_SSN_VALUE = 145014000;
+
+    public static Document SAMPLE_DOC = new Document()
+            .append("name", SAMPLE_NAME_VALUE)
+            .append("ssn", SAMPLE_SSN_VALUE)
+            .append("bloodType", "AB-")
+            .append("medicalRecords", Arrays.asList(
+                    new Document().append("weight", "180"),
+                    new Document().append("bloodPressure", "120/80")))
+            .append("insurance", new Document()
+                .append("policyNumber", 123142)
+                .append("provider", "MaestCare"));
+
+    // Reads the 96-byte local master key
+    public static byte[] readMasterKey(String filePath) throws IOException {
+        FileInputStream fis = null;
+        int numBytes = 96;
+        byte[] fileBytes = new byte[numBytes];
+
+        try {
+            fis = new FileInputStream(filePath);
+            if (fis.read(fileBytes) < numBytes)
+                throw new Exception("Expected to read 96 bytes from file");
+        } catch(Exception e) {
+            e.printStackTrace();
```

Comment:
Just let the exception be thrown rather than exiting

Reply:
Thanks, will fix.